pom.xml

```diff
@@ -77,7 +77,24 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-api-mockito</artifactId>
```

Comment:
I don't think this dep is needed either

Reply:
Yep, didn't need deeptestutils either turns out. Removed both.